php/www/event.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require_once("./lib/connect-db.php");
+require_once("./components/header/index.php");
+require_once("./components/sidebar/index.php");
+require_once("./components/article-card/index.php");
+require_once("./components/content-card/index.php");
+?>
+
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <title>学内掲示板</title>
+    <?php require_once('./lib/bootstrap.php'); ?>
+    <link rel="stylesheet" href="/style/main.css">
+</head>
+
+<body>
+    <div class="d-flex">
+        
+        <?php 
+            $sidebar = new Sidebar();
+            $sidebar->render();
+        ?>
+
+        <!-- メインコンテンツ -->
+        <div class="content w-100 ms-4">
+            <?php 
+                $header = new Header();
+                $header->render();
+            ?>
+
+            <section>
+            <h4 class="m-0 p-4 pb-0">最近の投稿</h4>
```

Comment:
`section`全体にpaddingを当てる

```suggestion
            <section class="p-4">
                <h4>最近の投稿</h4>
```